src/js/big-play-button.js

```diff
@@ -58,10 +59,8 @@ class BigPlayButton extends Button {
 
     const playFocus = () => playToggle.focus();
 
-    if (playPromise && playPromise.then) {
-      const ignoreRejectedPlayPromise = () => {};
-
-      playPromise.then(playFocus, ignoreRejectedPlayPromise);
+    if (isPromise(playPromise)) {
+      playPromise.then(playFocus, () => {});
```

Comment:
why not import `silencePromise` and use that?

Reply:
Copy/paste from my other PR, good call.

Reply:
Oh, actually, because it's not silencing it.

Reply:
It isn't? What the difference between `ignoreRejectedPlayPromise` and `silencePromise`?

Oh, I see. We are silencing the playPromise but we're also adding a `playFocus` handler.

Reply:
we could potentially do something like:
```js
if (isPromise(playPromise)) {
  silencePromise(playPromise);
  playPromise.then(playFocus);
}
```

Reply:
Seems like overkill...

Reply:
yeah, either way. using `silencePromise` is probably extra explicit/clear but either is fine.